README.md

```diff
@@ -1,2 +1,47 @@
-# lifion-check-dependencies
-Check that installed NPM modules are the latest-currently available version.
+# lifion-verify-deps
+
+Verifies that installed NPM modules are the latest currently available version.
+
+## Getting Started
+
+To install the module:
+
+```sh
+npm install lifion-verify-deps --global
+```
+
+To run command:
+
+```sh
+lifion-verify-deps
+```
+
+To use as module:
+
+```sh
+const verifyDeps = require('lifion-verify-deps');
+
+verifyDeps({ dir: './path-to/package.json' })
```

Comment:
Should `dir` also include `package.json`?

Reply:
Woops, you're right, should be to the directory...